autoload/neomake/makers/ft/bib.vim

```diff
@@ -0,0 +1,15 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#bib#EnabledMakers() abort
+    return []
+endfunction
+
+function! neomake#makers#ft#bib#bibtex() abort
+    return {
+                \ 'exe': 'bibtex',
+                \ 'args': ['-terse', '%:r'],
+                \ 'append_file': 0,
```

Comment:
So it acts on the directory?

Reply:
No, bibtex is kinda weird. To build a bibliography, you need to edit a file named FILENAME.bib but bibtex wants a file named either FILENAME or FILENAME.aux, hence I have to use %:r in the args and ask the maker not to provide the filename to bibtex.